overlay/tun_linux.go

```diff
@@ -36,6 +36,7 @@ type tun struct {
 	routeTree       atomic.Pointer[bart.Table[netip.Addr]]
 	routeChan       chan struct{}
 	useSystemRoutes bool
+	useSystemRoutesBuffer int
```

Comment:
```suggestion
	useSystemRoutesBufferSize int
```

sort of a nit, but I wouldn't want folks to think this was the actual buffer

Reply:
fixed